spec/unit/pops/lookup/context_spec.rb

```diff
@@ -171,13 +171,13 @@ module Lookup
 
       context 'and multiple compilations' do
 
-        before(:each) { Puppet.settings[:environment_timeout] = 'unlimited' }
         after(:each) do
           Puppet.settings[:environment_timeout] = 0
```

Comment:
FWIW, it shouldn't be necessary to reset the env timeout at the end of the test as we create a new settings object for each test (to avoid tests leaking state)